python/paddle/v2/framework/nets.py

```diff
@@ -103,22 +103,18 @@ def sequence_conv_pool(input,
                        filter_size,
                        pool_size,
                        pool_stride,
```

Comment:
`pool_size` and `pool_stride` are also useless. Please remove them.

Reply:
Done.